generation/anomaly_detection/2d_classifierfree_guidance_anomalydetection_tutorial.ipynb

```diff
@@ -0,0 +1,1033 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    n_iterations = 2e4
</pre><p>This variable should be called <code>max_epochs</code>. The CICD system will run this notebook with this value set to 2 so that training is fast. Further cells should take into account that this value may be what you give or 2 and so should expect that many items in any lists you accumulate during training. This should be changed for the other notebooks as well.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/1818/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>